client/StatusTable/saga.js

```diff
@@ -36,32 +18,33 @@ function* startSaga() {
         const rawWorkdays = yield call(getWorkdaysForMonth, _id);
 
         let workweeks = [];
-        let daysOfTheWeek = [];
-        let currentNumberOfTheWeek = rawWorkdays[0].numberOfTheWeek;
-        for (let i = 0; i < currentNumberOfTheWeek - 1; i++){
-            workweeks.push({
-                numberOfTheWeek: i + 1,
-                daysOfTheWeek: []
-            });
-        }
-        rawWorkdays.forEach(workday => {
-            if (workday.numberOfTheWeek === currentNumberOfTheWeek) {
-                daysOfTheWeek.push(workday);
-            } else {
+        if (rawWorkdays.length) {
```

Comment:
seems like you can use array.reduce there to follow more functional programming way and reduce code